fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/impl/networking/NetworkingImpl.java

```diff
@@ -16,27 +16,21 @@
 
 package net.fabricmc.fabric.impl.networking;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import net.minecraft.network.ClientConnection;
-import net.minecraft.network.NetworkState;
-import net.minecraft.network.PacketByteBuf;
 import net.minecraft.util.Identifier;
 
-import net.fabricmc.fabric.api.networking.v1.PacketByteBufs;
-import net.fabricmc.fabric.api.networking.v1.ServerLoginConnectionEvents;
-import net.fabricmc.fabric.api.networking.v1.ServerLoginNetworking;
-import net.fabricmc.fabric.api.networking.v1.ServerPlayNetworking;
-import net.fabricmc.fabric.mixin.networking.accessor.ServerLoginNetworkHandlerAccessor;
+import net.fabricmc.loader.api.FabricLoader;
 
 public final class NetworkingImpl {
 	public static final String MOD_ID = "fabric-networking-api-v1";
 	public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID);
+	/**
+	 * When enabled the fabric packet is written to the {@link net.minecraft.network.PacketByteBuf} on the calling thread.
+	 * This is enabled by default in a development environment to test the writing when in single player.
+	 */
+	public static final boolean WRITE_FABRIC_PACKET_CALLING_THREAD = Boolean.parseBoolean(System.getProperty("fabric-api.networking.write-fabric-packet-calling-thread", FabricLoader.getInstance().isDevelopmentEnvironment() ? "false" : "false"));
```

Comment:
Should this be `? "true" : "false"`?

Reply:
Opps yeah! forgot to change this back when debugging (I was too lazy to use the system prop I just added :D) I will fix this tomorrow (along with single player not working.

Reply:
As an addendum, I have somewhat mixed feelings about this. I'm definitely in favour of having extra checks in dev, but in this case it changes behaviour, which I feel iffy about.

That said, I don't really have any good alternative suggestions. In an ideal world, people would have unit tests for round-tripping packets (this sort of thing is perfect for property testing!), but I realise this is modded Minecraft we're talking about :).